tests/lib/tokenize.js

```diff
@@ -199,4 +199,20 @@ describe("tokenize()", () => {
         espree.tokenize("foo", Object.freeze({ ecmaFeatures: Object.freeze({}) }));
     });
 
+    /**
+     * Make sure we tokenize closing curly brace in a block statement
```

Comment:
Based on what you understand of the issue, would this be an improvement?

```suggestion
     * Make sure we tokenize closing curly brace in a block statement at end of file
```

Reply:
I think so

Reply:
added